.github/workflows/main.yml

```
@@ -20,6 +20,11 @@ jobs:
           python -m pip install tox
       - name: Run tests
         run: tox
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          functionalities: gcov
```

How about adding some markers?
```suggestion
          flags: >-
            GHA,
            ${{ runner.os }},
            ${{ matrix.python }}
          functionalities: gcov
```

---

Sure,I will add these as well.

---

FYI it's best to accept changes via GH UI because this way it properly marks discussion threads as resolved and also credits the suggester in the commit message as a co-author.

---

Thanks, I wasn't aware of this.

---

I see `functionalities: gcov` disables gcov. What is gcov and why does it need to be disabled?

---

My understanding is that gcov is a coverage format for code written in C. By default, codecov attempts scanning the disk for various formats because it doesn't know what technologies are present, I suppose this disables probing some formats.

https://en.wikipedia.org/wiki/Gcov